src/main/java/com/microsoft/jenkins/containeragents/aci/AciContainerTemplate.java

```diff
@@ -84,6 +86,7 @@ public AciContainerTemplate(String name,
                                 String image,
                                 String command,
                                 String rootFs,
+                                String ipType,
```

Comment:
unneeded breaking change to configuration, affects things like configuration as code.

Create a setter instead annotated with `DataBoundSetter`